src/overlays/actors/ovl_En_Poh/z_en_poh.c

```diff
@@ -96,123 +150,874 @@ static DamageTable D_80B2F6E0 = {
+    } else {
+        f32 rand = Rand_ZeroOne();
+
+        this->unk_198 = (s32)(rand * 30.0f) + 225;
+        this->unk_199 = (s32)(rand * 100.0f) + 155;
+        this->unk_19A = (s32)(rand * 160.0f) + 95;
+        this->unk_19B = 200;
+    }
+}
+
+void EnPoh_Update(Actor* thisx, GlobalContext* globalCtx2) {
+    EnPoh* this = THIS;
+    GlobalContext* globalCtx = globalCtx2;
+    s32 pad;
+
+    if (this->colliderSph.base.atFlags & AT_HIT) {
+        this->colliderSph.base.atFlags &= ~AT_HIT;
+        func_80B2D5DC(this);
+    }
+
+    func_80B2E438(this, globalCtx);
+    func_80B2E55C(this);
+    this->actionFunc(this, globalCtx);
+    Actor_SetVelocityAndMoveYRotationAndGravity(&this->actor);
+    if ((this->actionFunc == func_80B2CF28) && (this->unk_18E < 10)) {
+        this->actor.flags |= 0x1000000;
+        CollisionCheck_SetAT(globalCtx, &globalCtx->colChkCtx, &this->colliderSph.base);
+    }
+
+    Collider_UpdateCylinder(&this->actor, &this->colliderCylinder);
+    if (this->colliderCylinder.base.acFlags & AC_ON) {
+        CollisionCheck_SetAC(globalCtx, &globalCtx->colChkCtx, &this->colliderCylinder.base);
+    }
+
+    CollisionCheck_SetOC(globalCtx, &globalCtx->colChkCtx, &this->colliderCylinder.base);
+    CollisionCheck_SetOC(globalCtx, &globalCtx->colChkCtx, &this->colliderSph.base);
+    Actor_SetHeight(&this->actor, 42.0f);
+
+    if ((this->actionFunc != func_80B2D07C) && (this->actionFunc != func_80B2D628)) {
+        if (this->actionFunc == func_80B2DB44) {
+            this->actor.shape.rot.y = BINANG_ROT180(this->actor.world.rot.y);
+        } else {
+            this->actor.shape.rot.y = this->actor.world.rot.y;
+        }
+    }
+
+    func_80B2E8E0(this);
+    this->actor.shape.shadowAlpha = this->unk_197;
+    if (this->unk_298 > 0.0f) {
+        Math_StepToF(&this->unk_298, 0.0f, 0.05f);
+        if (this->unk_197 != 255) {
+            if (this->unk_197 * 0.003921569f < this->unk_197) {
+                this->unk_298 = this->unk_197 * 0.003921569f;
+            }
+        }
+
+        this->unk_29C = (this->unk_298 + 1.0f) * 0.225f;
+        this->unk_29C = CLAMP_MAX(this->unk_29C, 0.45f);
+    }
+}
+
+s32 EnPoh_OverrideLimbDraw(GlobalContext* globalCtx, s32 limbIndex, Gfx** dList, Vec3f* pos, Vec3s* rot, Actor* thisx,
+                           Gfx** gfx) {
+    EnPoh* this = THIS;
+
+    if ((this->unk_197 == 0) || (limbIndex == 18) || ((this->actionFunc == func_80B2D300) && (this->unk_18E >= 2))) {
+        *dList = NULL;
+    }
+
+    if (limbIndex == 19) {
+        gDPPipeSync((*gfx)++);
+        gDPSetEnvColor((*gfx)++, this->unk_194, this->unk_195, this->unk_196, this->unk_197);
+    }
+
+    return false;
+}
+
+void EnPoh_PostLimbDraw(GlobalContext* globalCtx, s32 limbIndex, Gfx** dList, Vec3s* rot, Actor* thisx, Gfx** gfx) {
+    static s8 D_80B2F71C[] = {
+        -1, -1, -1, -1, 4, 5, -1, -1, -1, 0, 1, -1, -1, -1, -1, -1, 2, -1, -1, 3, -1, 0, 0, 0,
+    };
```

Comment:
```suggestion
    static s8 D_80B2F71C[] = {
        -1, -1, -1, -1, 4, 5, -1, -1, -1, 0, 1, -1, -1, -1, -1, -1, 2, -1, -1, 3, -1,
    };
```
21 limbs, so these trailing zero's are padding